content/en/getting-started/security/_index.md

```diff
@@ -5,27 +5,34 @@ weight: 50
 ---
 
 {{% pageinfo %}}
-Trustgrid’s mission is to replace aging Edge connectivity solutions with innovative, software defined solutions that integrate security and compliance in a way impossible for legacy solutions, while improving efficiency in deployment and life-cycle management.
+Trustgrid’s mission is to replace aging Edge connectivity solutions with innovative, software-defined solutions that integrate security and compliance in a way impossible for legacy solutions while improving efficiency in deployment and life-cycle management.
 {{% /pageinfo %}}
 
-### Authentication
+## Authentication
 
-Pre-shared keys (PSK) are the most common method for authenticated connectivity and present significant risk when implemented poorly. Certificate based authentication is difficult and requires advanced skill sets. Trustgrid is the ‘root of trust’ in a Public Key Infrastructure (PKI) built by our security experts to enable automated certificate management across Trustgrid networks. All devices enroll in the PKI and are managed centrally from Trustgrid’s cloud infrastructure. Certificates are issued per device and are used to authenticate, and for encryption of all traffic.
+Pre-shared keys (PSK) are the most common method for authenticated connectivity and present a significant risk when implemented poorly. Certificate-based authentication is difficult and requires advanced skill sets. Trustgrid is the ‘root of trust’ in a Public Key Infrastructure (PKI) built by our security experts to enable automated certificate management across Trustgrid networks. All devices enroll in the PKI are managed centrally from Trustgrid’s cloud infrastructure. Certificates are issued per device and are used to authenticate, and for encryption of all traffic.
 
-### Authorization
+## Authorization
 
 Central to the security of the Trustgrid network is an authorization model derived from Google’s [Beyond Corp](https://cloud.google.com/beyondcorp) (aka Zero Trust from Forrester) initiative. This places an implicit deny on all traffic that cannot be configured to allow all traffic. Many breaches have been caused by allowing all traffic to overcome the burden of proper security configuration.
 
 Trustgrid uses a token to authorize communication between Trustgrid [nodes]({{<ref "/docs/nodes" >}}).
 
+## Encryption
+
 ### TLS Encryption
 
-All data on Trustgrid is encrypted from [node]({{<ref "/docs/nodes" >}}) to node using next generation encryption – TLS Mutual Authentication. The Internet Engineering Task Force (IETF) recommends TLS as the replacement to IPSec VPN.
+All data on Trustgrid is encrypted from [node]({{<ref "/docs/nodes" >}}) to node using next-generation encryption – TLS Mutual Authentication. The Internet Engineering Task Force (IETF) recommends TLS as the replacement to IPSec VPN. The TLS tunnels use the [TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 cipher](https://ciphersuite.info/cs/TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384/).
```

Comment:
Personally I'd leave out the next-generation encryption part, mutual TLS has been around a looong time

Reply:
I didn't write that but had the same thought